<div class="list-group list-group-flush">
    <a href=" https://pubmed.ncbi.nlm.nih.gov/23892401/" target="_blank" class="list-group-item" id="mylist">
        <div id="artical-img1"></div>
        <div id="article">
            Utilizing sequence intrinsic composition to classify protein-coding and long
            non-coding transcripts.
            <br><span class="mydate">🕛 2013 Jul 27</span>
        </div>

    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/28521017/" target="_blank" class="list-group-item" id="mylist">

        <div id="artical-img2"></div>
        <div id="article">
            CPC2: a fast and accurate coding potential calculator based on sequence
            intrinsic
            features.
            <br><span class="mydate">🕛 2017 Jul 3</span>
        </div>

    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/30649200/" target="_blank" class="list-group-item" id="mylist">
        <div id="artical-img3"></div>
        <div id="article">
            Characterization and identification
            of long non-coding RNAs based on feature relationship.
            <br><span class="mydate">🕛 2019 Sep 1</span>
        </div>
    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/25239089/" target="_blank" class="list-group-item" id="mylist">
        <div id="artical-img4"></div>
        <div id="article">
            PLEK: a tool for predicting long noncoding RNAs and messenger RNAs based on an
            improved k-mer scheme.
            <br><span class="mydate">🕛 2014 Sep 19</span>
        </div>

    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/23335781/" target="_blank" class="list-group-item" id="mylist">
        <div id="artical-img5"></div>
        <div id="article">
            CPAT: coding-potential assessment tool using an alignment-free logistic
            regression
            model.<br><span class="mydate">🕛 2013 Apr 1</span>
        </div>

    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/30753596/" target="_blank" class="list-group-item " id="mylist">
        <div id="artical-img6"></div>
        <div id="article">
            CPPred: coding potential prediction based on
            the global description of RNA sequence.
            <br><span class="mydate">🕛 2019 May 7</span>
        </div>

    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/29047334/" target="_blank" class="list-group-item" id="mylist">
        <div id="artical-img7"></div>
        <div id="article">
            A support
            vector machine based method to distinguish long noncoding RNAs from protein
            coding
            transcripts.
            <br><span class="mydate">🕛 2017 Oct 18</span>
        </div>

    </a>
    <a href="https://pubmed.ncbi.nlm.nih.gov/31484412/" target="_blank" class="list-group-item" id="mylist">

        <div id="artical-img8"></div>
        <div id="article">
            PredLnc-GFStack: a global
            sequence feature based on a stacked ensemble learning
            method for predicting lncRNAs from transcripts.
            <br><span class="mydate">🕛 2019 Sep 3</span>
        </div>


    </a>
    <a href="https://academic.oup.com/bioinformatics/article/34/22/3825/5021677?login=false" target="_blank"
        class="list-group-item" id="mylist">
        <div id="artical-img9"></div>
        <div id="article">
            an ab initio lncRNA
            identification and functional annotation tool based on deep
            learning.
            <br><span class="mydate">🕛 2018 May 29 </span>
        </div>
    </a>
</div>

<!-- <p class="mytitle">INTRODUCTION<br></p>
                <hr>
                <p>
                    With the development of high-throughput sequencing technology, numerous genomic transcript data have
                    been
                    discovered,
                    which promotes the study of genomics, especially the human
                    genome. However, it is worth noting that >90% of the transcripts
                    in the human genome are not encoded into proteins, well
                    known as noncoding RNAs (ncRNAs). Long noncoding RNAs,(long ncRNAs or lncRNAs) are a class of ncRNAs
                    with
                    >200 nucleotides in length.
                    According to the regions that lncRNAs are located and transcribed from, lncRNAs can be divided into
                    four
                    major categories: intergenic lncRNAs (those transcribed from intergenic regions), intronic lncRNAs
                    (those
                    transcribed entirely from introns of
                    protein-coding genes), sense lncRNAs (those transcribed from the sense strand of protein-coding
                    genes)
                    and
                    antisense lncRNAs(those transcribed from the antisense strand
                    of protein-coding genes). According to effects exerted on DNA sequences, lncRNAs can be
                    classified
                    into
                    two classes:
                    cis-lncRNAs that regulate the expression of genes in close genomic proximity and translncRNAs that
                    regulate
                    the expression of distant
                    genes.

                </p> -->
<!-- <img class="img-fluid myimg" src="/static/image/introduction.png" alt="introduction">
                <div class="imgPreview">
                    <img src="#" alt="" id="imgPreview">
                </div> -->

<!-- </div> -->

<div class="shadow mypage">
    <p class="mytitle">OUR WORK<br></p>
    <hr>


    <p><svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="#033a55b2" class="bi bi-check2-circle"
            viewBox="0 0 20 20">
            <path
                d="M2.5 8a5.5 5.5 0 0 1 8.25-4.764.5.5 0 0 0 .5-.866A6.5 6.5 0 1 0 14.5 8a.5.5 0 0 0-1 0 5.5 5.5 0 1 1-11 0z" />
            <path
                d="M15.354 3.354a.5.5 0 0 0-.708-.708L8 9.293 5.354 6.646a.5.5 0 1 0-.708.708l3 3a.5.5 0 0 0 .708 0l7-7z" />
        </svg> We introduce seven major databases with lncRNA
        resources, which are commonly used in lncRNA prediction.</p>


    <p><svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="#033a55b2" class="bi bi-check2-circle"
            viewBox="0 0 20 20">
            <path
                d="M2.5 8a5.5 5.5 0 0 1 8.25-4.764.5.5 0 0 0 .5-.866A6.5 6.5 0 1 0 14.5 8a.5.5 0 0 0-1 0 5.5 5.5 0 1 1-11 0z" />
            <path
                d="M15.354 3.354a.5.5 0 0 0-.708-.708L8 9.293 5.354 6.646a.5.5 0 1 0-.708.708l3 3a.5.5 0 0 0 .708 0l7-7z" />
        </svg> We classify all features into six categories: codonrelated features, ORF-related
        features,
        GC-related features, transcript-related features, coding sequence related features and
        structure-related features, and
        detailed descriptions and formulas of features are
        provided.</p>

    <p><svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="#033a55b2" class="bi bi-check2-circle"
            viewBox="0 0 20 20">
            <path
                d="M2.5 8a5.5 5.5 0 0 1 8.25-4.764.5.5 0 0 0 .5-.866A6.5 6.5 0 1 0 14.5 8a.5.5 0 0 0-1 0 5.5 5.5 0 1 1-11 0z" />
            <path
                d="M15.354 3.354a.5.5 0 0 0-.708-.708L8 9.293 5.354 6.646a.5.5 0 1 0-.708.708l3 3a.5.5 0 0 0 .708 0l7-7z" />
        </svg> We review all lncRNA prediction methods and divide
        them into three categories. The details of computational methods are described and analyzed.
    </p>

    <p><svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="#033a55b2" class="bi bi-check2-circle"
            viewBox="0 0 20 20">
            <path
                d="M2.5 8a5.5 5.5 0 0 1 8.25-4.764.5.5 0 0 0 .5-.866A6.5 6.5 0 1 0 14.5 8a.5.5 0 0 0-1 0 5.5 5.5 0 1 1-11 0z" />
            <path
                d="M15.354 3.354a.5.5 0 0 0-.708-.708L8 9.293 5.354 6.646a.5.5 0 1 0-.708.708l3 3a.5.5 0 0 0 .708 0l7-7z" />
        </svg> We provide a python package named ezLncPred to
        integrate existing state-of-the-art lncRNA prediction
        methods. Regardless of the runtime environment
        requirements of methods, users can apply different
        methods easily with ezLncPred.
    </p>
</div>

<!-- <p class="mytitle">关于网站<br></p>
                <hr>
                <p>
                    关于网站的主题内容及框架如图：
                    <img src="/static/image/introduction.png" style="max-width:100%; height:auto;"
                        class="img-responsive " alt="introduction">
                    <!-- <img src="/static/image/introduction.png" alt="introduction"> -->
· 本网站是lcnRNA预测网站，主要介绍lncRNA预测工具以及提供预测平台<br>
· 网站主要是用django + mysql + nginx 搭建，使用容器化部署方式在阿里云服务器上搭建。<br>
· 如有问题，可使用邮箱联系本网站开发人员：xinranxu@hzau.edu.cn
</p>
<p class="mytitle">关于作者<br></p>
<hr>
<p>
    If you have any comments or feedback, don’t hesitate to contact us. We’re always looking for new
    ideas about
    how to make the site better and more informative.
</p>
<p>
    Please direct your question to: Xinran Xu,
    <a href="mailto:xinranxu@webmail.hzau.edu.cn">xinranxu@webmail.hzau.edu.cn</a>

</p>
<p class="mytitle">功能介绍<br></p>
<hr>
<p>
    PredlncRNA提供一个lncRNA预测平台，在这里，你可以获取最新的lncRNA预测工具推送，可以选择九种不同的工具以及物种进行预测，
    并且保存结果。
</p> -->